book/src/template_syntax.md

```diff
@@ -658,6 +658,16 @@ E.g. to test if the least significant bit is set in an integer field:
 {% endif %}
 ```
 
+### Type conversion
+
+You can use the [`as`](https://doc.rust-lang.org/std/keyword.as.html) operator in `{{ … }}`
+expressions, and `{% … %}` blocks. It works the same as in Rust, but with some deliberate
+restrictions:
+
+- You can only use [primitive types](https://doc.rust-lang.org/std/primitive/index.html)
+  like `i32` or `f64` both as source variable type and as target type.
+- If the source is a reference to a primitive type, e.g. `&&&bool`, then rinja automatically
```

Comment:
Still forgot backticks around `rinja`. :p

We do it in multiple places so for the power of coherency, we need to continue! 🤣

Reply:
But "rinja" is a name that we don't use as a code snippet in here. I would rather remove the backticks around the other instances in book/src/template_syntax.md:299 and README.md:3. Maybe replace it with `**Rinja**` in the book (start of a sentence; bold like the first use of a lemma in Wikipedia)?

Reply:
We need to pick one, I'm fine with whichever.